test/parallel/test-vm-module-basic.js

```diff
@@ -63,3 +64,13 @@ const { SourceTextModule, createContext } = require('vm');
   const m3 = new SourceTextModule('3', { context: context2 });
   assert.strictEqual(m3.url, 'vm:module(0)');
 })();
+
+// Check inspection of the instance
+(() => {
+  const context = createContext({ foo: 'bar' });
+  const m = new SourceTextModule('1', { context });
+
+  // eslint-disable-next-line max-len
```

Comment:
Nit: I would rather wrap the line into multiple ones instead of disabling eslint here.